### ISSUE TITLE
Toolcore not welding projections

### ISSUE THREAD

**Issue by Lutavian** [DraconisCluster/DX-Issues#43]
** tl; dr **
Reported that Toolcore is not welding up projections. According to the report the following was tried before making the ticket:
Fixship the grid w/ welders
fixship the grid that has the projection
replace welders
F5
restart game
removed client side plugins
pc restart
Handwelding worked
non-toolcore worked

** Full info **
Ticket 50117 https://discord.com/channels/516135382191177728/1225824663855173662
"Toolcore welders are not welding up projections on-server. I reported on the toolcore discord already, they think something is interacting server-side potentially a plugin. Just wanted to get a ticket open here for documentation and (hopefully) some debugging. I had Raz test and he had the same issue as well, I haven't been able to convince anyone else to test on server. It works fine on the creative save."

**Screenshots**
Video - https://cdn.discordapp.com/attachments/1225824663855173662/1225825485422985266/no-weld.mp4?ex=662289f2&is=661014f2&hm=c550463b6a85d9b9da0a12a50eee68d6289b1686e90fe4886613bd3e6f2be1e3&

**Comment by draconb** [comment]
This should be fixed up now was an issue with TC / MGP updates.

**Comment by draconb** [comment]
Fixed with external updates
